.github/renovate.json5

```diff
@@ -64,6 +68,21 @@
       ],
       "allowedVersions": "<1.0"
     }, {
+// We don't want digest updates for kubernetes dependencies, as they are too
+// noisy
+      "matchDatasources": [
+        "go"
+      ],
+      "groupName": "kubernetes patches",
+      "matchUpdateTypes": [
+        "digest",
```

Comment:
Does this mean that we will get non-release versions of k8s deps?

Reply:
No, it's just for packages that do not use tags, such as k8s.io/utils. Renovate would open a PR every time a new commit gets on main, this way it's just going to do so once a month.

Reply:
Yes, it should, but given that we are using distroless images I expect only scheduled PRs 🤞